modules/metallb-installing-using-web-console.adoc

```diff
@@ -16,7 +16,9 @@ As a cluster administrator, you can install the MetalLB Operator by using the {p
 
 . In the {product-title} web console, navigate to *Operators* -> *OperatorHub*.
 
-. Search for the MetalLB Operator, then click *Install*.
+. Scroll or type a keyword into the *Filter by keyword* box to find the Operator you want. For example, type `metallb` to find the MetalLB Operator.
+
+  You can also filter options by *Infrastructure Features*. For example, select *Disconnected* if you want to see Operators that work in disconnected environments, also known as restricted network environments.
```

Comment:
This if we follow exactly what is on the UI and I think we should be written as *Infrastructure features*. I have an additional question open on this as when I select *Disconnected* MetalLB will not come up in the search output. Maybe metadata describing MetalLB needs to be updated.

Reply:
Hi @kquinn1204 I committed the new changes as you suggested, 

Regarding the questions: when I select Disconnected MetalLB will not come up in the search output. 
>> I tested this in my environment, and I see that when I select Disconnected environment and search for MetalLB, It doesn't show up for 4.9 as well as 4.11.